test/src/main/java/org/apache/accumulo/test/ShellIT.java

```diff
@@ -236,6 +236,17 @@ public void insertDeleteScanTest() throws IOException {
     exec("delete \\x90 \\xa0 \\xb0", true);
     exec("scan", true, "\\x90 \\xA0:\\xB0 []    \\xC0", false);
     exec("deletetable test -f", true, "Table: [test] has been deleted");
+    // Add tests to verify use of --table parameter
+    exec("createtable test2", true);
+    exec("notable", true);
```

Comment:
What is 'notable' here?

Reply:
It verifies that one can call the 'notable' command without error.

Reply:
Oh, sorry. I guess I didn't realize we had such a command. I'm guessing you use that to change to a context without a table before proceeding with the test with the table arguments.